### PR TITLE
Fix debug build: fyit_stream does not have member chunk

### PR DESCRIPTION
**Pull request by biojppm**
This fails to compile when in debug mode. Also, added missing file fy-docbuilder.c to CMakeLists.txt

### CODE REVIEW

File: src/lib/fy-dump.c [hunk 1]
```diff
@@ -293,8 +293,7 @@ void fyp_debug_dump_input(struct fy_parser *fyp, const struct fy_input_cfg *fyic
 		break;
 	case fyit_stream:
 		fyp_scan_debug(fyp, "%s: stream=\"%s\" fileno=%d chunk=%zu\n", banner,
-				fyic->stream.name, fileno(fyic->stream.fp),
```
Comment (pantoniou) [inline src/lib/fy-dump.c:296]: The chunk is still present in the printf string; please remove and update the PR.

Comment (biojppm) [reply src/lib/fy-dump.c:296]: Indeed. It was already removed in a subsequent commit.

File: src/lib/fy-dump.c [hunk 2]
```diff
@@ -292,9 +292,8 @@ void fyp_debug_dump_input(struct fy_parser *fyp, const struct fy_input_cfg *fyic
 				fyic->file.filename);
 		break;
 	case fyit_stream:
-		fyp_scan_debug(fyp, "%s: stream=\"%s\" fileno=%d chunk=%zu\n", banner,
-				fyic->stream.name, fileno(fyic->stream.fp),
-				fyic->stream.chunk);
+		fyp_scan_debug(fyp, "%s: stream=\"%s\" fileno=%d\n", banner,
```
Comment (biojppm) [inline src/lib/fy-dump.c:295]: @pantoniou note the chunk is not here

Comment (pantoniou) [reply src/lib/fy-dump.c:295]: Ugh, yeah, you're right; serves me right for commenting on a phone...

**Pull request merged**